apps/vscode/src/main.ts

```diff
@@ -205,7 +209,7 @@ function manuallySelectWorkspaceDefinition() {
             'nxWorkspacePath',
             selectedDirectoryRelativePath,
           );
-          updateMcpServerWorkspacePath(selectedDirectory);
+          await selectedDirectory;
```

Comment:
is `selectedDirectory` a Promise? looks like a string. Maybe it should be

```
await updateMcpServerWorkspacePath(selectedDirectory);
```
?